cloud/pkg/cloudhub/config/config.go

```diff
@@ -0,0 +1,139 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"k8s.io/klog"
+
+	"github.com/kubeedge/beehive/pkg/common/config"
+)
+
+var c Configure
+var once sync.Once
+
+type Configure struct {
+	ProtocolWebsocket  bool
+	ProtocolQuic       bool
+	ProtocolUDS        bool
+	MaxIncomingStreams int
+	Address            string
+	Port               int
+	QuicPort           int
+	UDSAddress         string
+	KeepaliveInterval  int
+	Ca                 []byte
+	Cert               []byte
+	Key                []byte
+	WriteTimeout       int
+	NodeLimit          int
+}
+
+func InitConfigure() {
+	once.Do(func() {
+		var errs []error
+		defer func() {
+			if len(errs) != 0 {
+				for _, e := range errs {
+					klog.Errorf("%v", e)
+				}
+				klog.Error("init cloudhub config error")
+				os.Exit(1)
+			} else {
+				klog.Infof("init cloudhub config successfully，config info %++v", c)
+			}
```

Comment:
Remove the defer and move lines behind #115.  And no need else here.

Reply:
ok